### ISSUE TITLE
Kodi 18 standalone

### ISSUE THREAD

**Issue by CyberLine** [BrosMakingSoftware/Kodi-Launches-EmulationStation-Addon#6]
Hi there,

today i tried the Plugin on Kodi 18 RC3.
It starts emulationstation in front of kodi, but wont accept any controller input, maybe because K18 is running behind and is detecting the Controller itself?

Is there anything i can provide in logs?

**Comment by dommtardif** [comment]
Good morning, could you give us more info on your setup please? I looked for kodi standalone but didn't find much info, is that a specific Linux distro? I'd like to reproduce the bug but am not sure what you have.

**Comment by CyberLine** [comment]
No, it's not distro related. You can boot any distro directly to kodi
instead of x11. Some pre build images like libreelec, openelec use this
method.

Dominique Tardif <notifications@github.com> schrieb am Do., 27. Dez. 2018,
14:33:

**Comment by dommtardif** [comment]
Thanks for the info. I'll have a look when I have a chance

**Comment by mivalsten** [comment]
Hi,
Bumping this issue as I have simmilar problem.

**Comment by ArthurBorsboom** [comment]
I encounter the same issue.

EmulationStation starts nicely in front of Kodi. The input, both keyboard and gamepads, are not responding.
I have noticed that the gamepad input ends up in Kodi, which is behind EmulationStation.

System setup

- OS: Arch linux
- No window manager; just plain Xorg
- Not running root, but as an unprivileged user 'kodi'.

I am wondering if it has something to do with running Kodi without a window manager. Because of this EmulationStation does not get the input handed over from Kodi.

Also setting the value below in es_settings.cfg does not solve the issue for me, which has been suggested if you run EmulationStation without WM.

`<bool name="BackgroundJoystickInput" value="true" /> 
`

Kodi and emulationstation packages from the Arch Linux package repository

- kodi 18.7.1-5
- kodi-addon-game-libretro 1.1.1-7
- kodi-addon-inputstream-adaptive 2.4.5-1
- kodi-addon-peripheral-joystick 1.4.7-18
- kodi-platform 20190726.809c5e9-12
- kodi-standalone-service 1.103-1
- kodi-x11 18.7.1-5
- emulationstation 2.9.3-2

Some of the processes while only running Kodi are the following.

```
kodi        2821  0.0  0.0   3860  2248 tty1     Ss+  23:10   0:00 /usr/bin/xinit /usr/bin/kodi-standalone -- :0 -nolisten tcp vt1
kodi        2830  0.0  0.8 778612 67072 tty1     Sl   23:10   0:00 /usr/lib/Xorg :0 -nolisten tcp vt1
kodi        2840  0.0  0.0   7168  3432 tty1     S    23:10   0:00 /bin/sh /usr/bin/kodi-standalone
kodi        2844  0.0  0.0   7300  3544 tty1     S    23:10   0:00 /bin/sh /usr/bin/kodi --standalone
kodi        2850  6.4  2.3 2123640 184896 tty1   Sl   23:10   0:18 /usr/lib/kodi/kodi-x11 --standalone
```

The new processes while running EmulationStation in front of Kodi.

```
kodi       16531  0.3  0.1  18924 10432 tty1     S    23:26   0:00 /usr/bin/xterm -fullscreen -e /usr/bin/emulationstation
kodi       16532 39.2  2.7 736188 222684 pts/2   Ssl+ 23:26   0:05 /usr/bin/emulationstation
```

**Comment by ArthurBorsboom** [comment]
So in essence the screen is visible and fullscreen, but the focus is wrong.

**Comment by qirtaiba** [comment]
I have the same problem. When I run emulationstation in an XFCE desktop, the controllers and keyboard are detected. When I try to do it in Kodi, it is not. Any fix for this?
